Including pep658 in the simple API description (#1061)
pypa/packaging.python.org: source/specifications/simple-repository-api.rst

The current interface for querying available package versions and
retrieving packages from an index server is defined in :pep:`503`,
with the addition of "yank" support (allowing a kind of file deletion)

> as defined in :pep:`592`, specifying the interface version provided
> by an index server in :pep:`629` and optionally providing metadata 
> separated from the distrobution package as stated in :pep:`658`.

Reviewer: How about something like this?

```suggestion
in :pep:`592`, specifying the interface version provided
by an index server in :pep:`629`, and retrieving packaging metadata 
in :pep:`658`.
```

Author: A lot better!
I do question,  with it being so simple, does it properly reflect the scope of PEP 658?
Also, aren't the other PEP summaries given from a server-side perspective, and you describe PEP 658 from a client side's perspective (receiving instead of providing)?
